src/NHibernate/Cache/CacheFactory.cs

```diff
@@ -60,7 +60,7 @@ public static class CacheFactory
 		public static ICacheConcurrencyStrategy CreateCache(string usage, CacheBase cache)
 		{
 			if (log.IsDebugEnabled())
-				log.Debug("cache for: {0} usage strategy: {1}", cache.RegionName, usage);
+				log.Debug("cache for: {0} usage strategy: {1}", cache?.RegionName, usage);
```

Comment:
Creating a strategy with a `null` cache is very likely not expected, but still, it does "work", unless debug logs are enabled.
It would be better to unconditionally throw in case of null, but that would be a possible breaking change, so it should not be done in a 5.3.x